src/main/kotlin/no/skatteetaten/aurora/boober/feature/RouteFeature.kt

```diff
@@ -167,17 +175,19 @@ class RouteFeature(@Value("\${boober.route.suffix}") val routeSuffix: String) :
         val defaultAnnotations = adc.getRouteAnnotations("$ROUTE_DEFAULTS_FEATURE_FIELD/annotations/")
 
         val shouldGenerateAzureRoute = adc.isAzureConfigured()
-        val isFullyQualifiedHost = adc.isMsDnsCnameEnabled()
+        val isFullyQualifiedHost = adc.isMsDnsCnameEnabled() || adc.isAzureConfigured()
```

Comment:
Blir dette riktig? Jeg er usikker selv. Her blir ruta som ikke er av type=azure også satt til fullyQualifiedHost. Er det noe man vil?

Reply:
Er det en test case som mangler her, kanskje?

Endringen vil vel bare treffe ruter som IKKE har vanlig cname, men har azure-enabled? Litt usikker på hva som er vanlig bruksmønster her...

Reply:
Det vil jeg nok tro. Jeg har kikket litt gjennom koden og har vel egentlig kommet fram til at det er litt inkonsistent bruk av det feltet. Det feltet er legacy vil jeg påstå og har kun vært for å støtte opp under OCP4 før vi hadde CName. Det er vel kun dok som bruker det og de kunne vel gått over til cname for de vil jeg tro. Så vi burde egentlig prøve å få fjernet det etter min mening. @stromland  hva tenker du? 

Sånn det står nå blir det utifra en ConfiguredRoute generert to Openshift Routes, gitt at azure for routen er konfigurert. Den ene ruten har ikke Cname, mens ruten med label type=azure har cname. Den koden du har nå sier at den første ruten er en fullyQualifiedHost. Jeg tenker kanskje at det blir feil, men er ikke 100% enig med meg selv. Det har noe å si for litt validering og modify i forhold til ROUTE_URL som blir lagt på som env. 
For den første ruten vil du ende opp med en env med ROUTE_URL uten routeSuffix utifra hva jeg leser i koden. Det har man jo lyst på

Reply:
@stromland, @bj0rn: Burde jeg endre noe her? Jeg ser ikke helt hvordan feltet brukes, men tanken er at jeg ikke endrer noe for de som ikke har Azure enabled.